@xen-orchestra/web-core/lib/components/state-hero/VtsLoadingHero.vue

```diff
@@ -1,13 +1,51 @@
 <template>
-  <VtsStateHero :type busy>
-    {{ $t('loading-in-progress') }}
-  </VtsStateHero>
+  <div class="vts-loading-hero">
+    <VtsStateHero :type busy>
+      {{ $t('loading-in-progress') }}
+    </VtsStateHero>
+    <div class="content">
+      <div v-if="slots.title" class="title" :class="typoClass">
+        <slot name="title" />
+      </div>
+      <div v-if="slots.text" class="text typo-body-bold">
+        <slot name="text" />
+      </div>
+    </div>
```

Comment:
Be careful, this section will always be rendered, even if the slots aren’t used.